.circleci/config.yml

```diff
@@ -56,6 +56,8 @@ jobs:
           command: |
             cp .env.integration-tests .env
             make refresh SKIP_DUMMY=1 PYTEST_SENTRY_DSN=$PYTEST_SENTRY_DSN CIRCLECI=$CIRCLECI up_prod_detached integration_test_nimbus PYTEST_ARGS="$PYTEST_ARGS"
+      - store_artifacts:
+          path: ~/experimenter/experimenter/tests/integration/test-reports/report.htm
```

Comment:
Should these be `.html` instead of `.htm`?

Reply:
Wow that's some old school stuff.  File type suffixes were generally capped at 3 characters on old operating systems like Windows 95/98, so `.htm` and `.html` were interchangeable for a long time.  Hasn't been that way in decades but wow that takes me back.